doc/build-osx.md

```diff
@@ -17,6 +17,7 @@ macOS comes with a built-in Terminal located in:
 
 The Xcode Command Line Tools are a collection of build tools for macOS.
 These tools must be installed in order to build Bitcoin Core from source.
+You do not need to install all of Xcode.
```

Comment:
Seems weird to say "you don't need to install" and the next line "To install, run...". If we want to advertise something, I prefer something like: "Recent MacOS versions may contain Xcode by default"

Reply:
macOS does not ship with all of Xcode, but they do ship with the `xcode-select` command.